core/master/src/main/java/org/drftpd/master/GlobalContext.java

```diff
@@ -104,7 +104,7 @@
 
     private static AsyncThreadSafeEventService eventService = new AsyncThreadSafeEventService();
 
-    private static Set<Method> hooksMethods;
+    private static Set<Method> _hooksMethods;
```

Comment:
Underscore naming in Java doesnt make sens.

Reply:
Removed the underscore